docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -21,180 +18,13 @@ aliases:
   - /installation/viam-micro-server-dev/
 ---
 
-[The Micro-RDK](https://github.com/viamrobotics/micro-rdk/) is the lightweight version of Viam's full Robot Development Kit (RDK), designed for resource-limited embedded systems (ESP32 microcontrollers) that cannot run the fully-featured `viam-server`.
-`viam-micro-server` is the pre-built firmware built from the Micro-RDK and a default set of {{< glossary_tooltip term_id="module" text="modules" >}}, provided for convenience.
-If you want to use different modules on your microcontroller, you can build your own firmware with the Micro-RDK and your choice of modules using the instructions on this page.
+If no existing modules support your hardware or software, you can create your own.
```

Comment:
[q] the more I look at this sentence, the more I wonder if the framing is right. Some users might want to create a custom module even when their hardware/software is already supported by an existing module, right? I don't want to repeat the page description, but for an intro, maybe we should frame this in two ways:

* some users want to extend micro-rdk support to unsupported hardware (this case)
* some users want to write their own module to implement custom behavior on supported hardware

Reply:
Mmmm this gets into Viam philosophy a bit and applies not just to the Micro-RDK but certainly to modules in general.

One way to look at it is (sorry in advance for your nemesis scare quotes :D ): A module for a given bit of hardware should be good and enable all the things for that hardware. If that's not the case, it doesn't actually "support" your hardware. And if you want to add some layer of abstraction, that is "unsupported software"

Reply:
I should say, you do make a valid point, I just don't think we need to get into that level of semantics in this spot/in this PR! I'd rather keep it simple to avoid confusing the majority of users.